src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -7039,7 +7039,9 @@ namespace SurfaceGeometry {
             if (Item1 == 0 && state.dataHeatBal->NumOfZoneLists > 0)
                 ZLItem = UtilityRoutines::FindItemInList(state.dataIPShortCut->cAlphaArgs(3), state.dataHeatBal->ZoneList);
             if (Item1 > 0) {
-                ++NumIntMassSurfaces;
+                if (state.dataIPShortCut->lAlphaFieldBlanks(4)) {
```

Comment:
Only increment the number of surfaces if the field is blank.  I'm not sure I get why, I'll ask separately.